pkg/reconciler/v1alpha1/revision/table_test.go

```diff
@@ -493,10 +511,8 @@ func TestReconcile(t *testing.T) {
 			kpa("foo", "activate-revision", "Reserve", "busybox"),
 			// The Deployments match what we'd expect of an Reserve revision.
 			deploy("foo", "activate-revision", "Reserve", "busybox"),
-		},
-		WantCreates: []metav1.Object{
```

Comment:
What happened here?

Reply:
When we switched to KPA we started keeping around the service for Reserve Revisions